.github/test_e2e_single_chain_op_succinct_args.json

```diff
@@ -17,10 +17,23 @@
     "agglayer_prover_primary_prover": "mock-prover",
     "l1_seconds_per_slot": 2,
     "additional_services": [
-      "bridge_spammer" 
-    ]
+      "bridge_spammer"
+    ],
+    "zkevm_rollup_chain_id": 20201
```

Comment:
zkevm_rollup_chain_id - why are we hardcoding this?
shouldnt this be from kurtosis only?

Reply:
The `zkevm_rollup_chain_id` and `optimism_package.chains.network_params.network_id` parameters need to match, so I'm thinking it is better to be explicit here.

But I can try to tweak the args files and see what is the minimal set of params required for the tests to run succesfully.